include/drivers/pcie/msi.h

```diff
@@ -17,6 +17,23 @@
 extern "C" {
 #endif
 
+#ifdef CONFIG_PCIE_CONTROLLER
+struct generic_msi_vector {
+	unsigned int irq;
+	uint32_t address;
+	uint16_t eventid;
+	unsigned int priority;
+};
+
+typedef struct generic_msi_vector arch_msi_vector_t;
+
+#define PCI_DEVID(bus, dev, fn)  ((((bus) & 0xff) << 8) | (((dev) & 0x1f) << 3) | ((fn) & 0x07))
+#define PCI_BDF_TO_DEVID(bdf)	PCI_DEVID(PCIE_BDF_TO_BUS(bdf), \
+					  PCIE_BDF_TO_DEV(bdf),	\
+					  PCIE_BDF_TO_FUNC(bdf))
+
```

Comment:
A comment telling why you don't follow DTS representation (because it chose to follow x86's)

Reply:
Here it's not about the DT BDF, but the RequesterID format I use to pass to the GIC ITS DeviceID, I took the DEVID name from Linux... but maybe REQID should be better ?

Reply:
keep it as it is.